e2e/packages/contracts/mud.config.ts

```diff
@@ -60,9 +60,10 @@ export default defineWorld({
   },
   modules: [
     {
-      name: "Unstable_CallWithSignatureModule",
+      name: "CallWithSignatureModule",
```

Comment:
`name` isn't really tied to anything when using `artifactPath`, except the label that gets printed in log messages

not sure if we should make the types reflect this better or leaving a field here for a "nice name" is nice?